pkg/resource/firewall/hook.go

```diff
@@ -27,3 +36,251 @@ var (
 		ackrequeue.DefaultRequeueAfterDuration,
 	)
 )
+
+const ErrSyncingPutProperty = "Error syncing property '%s'"
```

Comment:
nit: this can become a `errors.New("Error syncing property")` and we can just use something like `fmt.Errorf("%v: %v", err, ErrSyncingPutProperty`)

Reply:
done